makefiles/pexec.mk

```
@@ -0,0 +1,43 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+SUBPROJECTS   += pexec
+PEXEC_VERSION := 1.0rc8
+DEB_PEXEC_V   ?= $(shell $(SED) 's/rc/~rc/g' <<< $(PEXEC_VERSION))
+
+pexec-setup: setup
+	wget -q -nc -P$(BUILD_SOURCE) http://ftp.gnu.org/gnu/pexec/pexec-$(PEXEC_VERSION).tar.gz
+	$(call EXTRACT_TAR,pexec-$(PEXEC_VERSION).tar.gz,pexec-$(PEXEC_VERSION),pexec)
```

Verify the integrity of the tarball (since you should download the signature)

```suggestion
	$(call PGP_VERIFY,pexec-$(PEXEC_VERSION).tar.gz)
	$(call EXTRACT_TAR,pexec-$(PEXEC_VERSION).tar.gz,pexec-$(PEXEC_VERSION),pexec)
```